### DISCUSSION TITLE
AttributeError: 'NoneType' object has no attribute 'shape'

### DISCUSSION THREAD

**Discussion by Snowej** [Mikubill/sd-webui-controlnet#132] [Q&A]
I was using the extension fine on release, but after the last few updates, whenever I try to generate an image I getting this error:

`AttributeError: 'NoneType' object has no attribute 'shape'`

Wondering what it means and how to fix it.

Thanks.

**Comment by ljleb** [comment]
related: https://github.com/Mikubill/sd-webui-controlnet/issues/110https://github.com/Mikubill/sd-webui-controlnet/issues/110

Seems like a recent commit fixed the issue. Can you try this @Snowej ?

**Comment by roseyu61** [comment]
I need help too!! I am in the same trouble now.
